src/ring_theory/chain_of_divisors.lean

```diff
@@ -110,7 +110,7 @@ begin
   { contradiction },
   obtain ⟨i, rfl⟩ := h₂.1 (dvd_trans hp' hr),
   refine congr_arg c (eq_of_ge_of_not_gt _ $ λ hi, _),
-  { rw [fin.le_iff_coe_le_coe, fin.coe_one, nat.succ_le_iff, ← fin.coe_zero,
+  { rw [fin.le_iff_coe_le_coe, fin.coe_one, nat.succ_le_iff, ← @fin.coe_zero (n.succ + 1),
```

Comment:
`fin.coe_zero` should definitely take an explicit argument for `n` now; as should `coe_one`